packages/opentelemetry-resources/src/platform/browser/detect-resources.ts

```diff
@@ -14,13 +14,11 @@
  * limitations under the License.
  */
 
-import { Resource } from '../../Resource';
-
 /**
  * Detects resources for the browser platform, which is currently only the
  * telemetry SDK resource. More could be added in the future. This method
  * is async to match the signature of corresponding method for node.
+ *
+ * @param config Configuration for resource detection
```

Comment:
If the function definition is moved, their inline doc comments should be moved too.

Reply:
removed